app/src/main/java/com/android/sample/ui/createRecipe/ChefImage.kt

```diff
@@ -0,0 +1,27 @@
+package com.android.sample.ui.createRecipe
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.offset
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.android.sample.R
+import com.android.sample.resources.C.Tag.CHEF_IMAGE_CORNER_RADIUS
+
+@Composable
```

Comment:
Hey, thanks a lot for the review! 😊

- For OffsetX and OffsetY, I actually added them to avoid hardcoding the image position, but since the wiki update on using `.weight()`, I don’t really need them anymore, so I’ll likely remove them.
- As for the rounded corners, there’s no particular reason at this point. Initially, it was to create a smoother look when overlapping with the button. Overlapping shouldn’t happen anymore, but do you think it’s worth keeping the corners rounded? I feel it doesn’t hurt to keep it as-is.

Reply:
If it's not needed we should remove them.